### ISSUE TITLE
line info currentPower shows only 1/3 of the real consumption

### ISSUE THREAD

**Issue by b2un0** [johnwoo-nl/emproto#3]
hi there,

it looks like, that `currentPower` only shows the the power of the first phase.

in the EVSEMaster App it shows me `10,27`, the `currentPower` shows only `3491`.

if i calculate all three lines like this

```javascript
let line = evse.getLines().get(1);

line.l1Power = line.l1Voltage * line.l1Electricity;
line.l2Power = line.l2Voltage * line.l2Electricity;
line.l3Power = line.l3Voltage * line.l3Electricity;
line.totalPower = line.l1Power + line.l2Power + line.l3Power;
```

i got the real value which match the app state.

is this a bug?

**Comment by b2un0** [comment]
jepp, it shows only l1

```json5
{
    "lineId": 1,
    "currentPower": 3471, // only l1
    "currentAmount": 1307.06,
    "l1Voltage": 229,
    "l1Electricity": 15.16,
    "l2Voltage": 230,
    "l2Electricity": 14.68,
    "l3Voltage": 229,
    "l3Electricity": 14.63,
    "innerTemp": 34.94,
    "outerTemp": 34.94,
    "currentState": "CHARGING",
    "gunState": "CONNECTED_LOCKED",
    "outputState": "CHARGING",
    "errors": [],
    "l1Power": 3471.64, // matches currentPower
    "l2Power": 3376.4,
    "l3Power": 3350.27,
    "totalPower": 10198.31 // matches the App value
}
```

**Comment by johnwoo-nl** [comment]
I'd say this is partly a bug in the OEM comms board (it actually reports this power field with this value; I'm just passing it through) and partly a misinterpretation of mine: I saw 3 voltage and amperage fields, and just 1 power field, so I assumed it was the power across all phases. Turns out it isn't; I checked the decompiled OEM app and it also does the same calculation as you (multiplying the volts and amps per phase and summing them) for the power display field. I didn't notice that because my wallbox is still connected on 1 phase only, so it accidentally was the correct value here.

I'll fix this in the EmEvseLine: I'll keep using the same currentPower field to preserve the API, but using the summed phase powers as the field value (and ignoring the field we get from the wallbox; or maybe I'll use the maximum of both, in case there are wallboxes which do report it but no per-phase info).
